docs/about/compatibility/openmp.md

```diff
@@ -461,14 +461,102 @@ To enable the generation of specialized kernels, follow these guidelines:
+To support the optimal use of shared memory on MI300A, the ROCm OpenMP compiler and runtime are optimized to use “zero-copy” also known as “no-copy” behavior as the default execution model where the CPU and GPU threads use the same addresses to access the same physical storage. The zero or no-copy behavior also ensures that the OpenMP map constructs do not result in device memory allocation and copies between host and device memory, as there is no physical device storage.
+
+For demonstration purposes, we are using [vec_add.cpp](#sample-program) in the examples below.
+
+To enable zero-copy behavior by default in the programs running on MI300A, follow these options:
+
+* When using `unified_shared_memory` pragma in the program:
+
+You can enable `unified_shared_memory` in your program using two options. One is to use `#pragma omp requires unified_shared_memory` construct in your source files and the other option is to use the compiler option `-fopenmp-force-usm` that enables `unified_shared_memory` without having to write it in the program. Irrespective of the option chosen to enable `unified_shared_memory`, you must build and run the program in an Xnack-enabled environment.
+
+Note that this implementation is already available for discrete GPU systems such as MI200.
+
+See how to build a program with `unified_shared_memory` pragma and `xnack+` target feature:
+
+```bash
+clang++ -fopenmp -offload-arch=gfx942:xnack+ -BUILD_WITH_USM vec_add.cpp -o vec_add
+```
+
+You can also build using `xnack-any` which is the default when an Xnack target feature is not explicitly specified:
+
+```bash
+clang++ -fopenmp -offload-arch=gfx942 -BUILD_WITH_USM vec_add.cpp -o vec_add
+```
+
+Note that the `-BUILD_WITH_USM` macro has been defined in [vec_add.cpp](#sample-program) to enable `#pragma omp requires unified_shared_memory`.
+
+Alternatively, see how to build using the ROCm compiler option `-fopenmp-force-usm`:
+
+```bash
+clang++ -fopenmp -offload-arch=gfx942 -fopenmp-force-usm vec_add.cpp -o vec_add
+```
+
+Execute the above-compiled program with Xnack-enabled as shown below:
+
+```bash
+HSA_XNACK=1 ./vec_add
+```
+
+* When not using `unified_shared_memory` pragma in the program:
+
+Build and run the program in an Xnack-enabled environment with `OMPX_APU_MAPS` environment variable set to 1 during execution. `OMPX_APU_MAPS` indicates to the OpenMP runtime that the system is an Accelerated Processing Unit (APU) and enables zero-copy mode.
+
+See how to build a program that does not contain the `unified_shared_memory` pragma:
+
+<!-- markdownlint-disable no-blanks-blockquote -->
+> * Using `xnack-any`
+
+> `clang++ -fopenmp -offload-arch=gfx942 vec_add.cpp -o vec_add`
+
+> * Using `xnack+`
+
+> `clang++ -fopenmp -offload-arch=gfx942:xnack+ vec_add.cpp -o vec_add`
+<!-- markdownlint-enable no-blanks-blockquote -->
```

Comment:
@SwRaw 
Added indentation via blockquotes (>) here

Markdown does not support indents well otherwise

Reply:
> @SwRaw Added indentation via blockquotes (>) here
> 
> Markdown does not support indents well otherwise

Thanks, @samjwu. Is this in our Contributing guide? @LisaDelaney

Reply:
Likely not. I was also not aware of this until I searched for it.

Reply:
That's good to know, Sam. Thanks for the update. @LisaDelaney could we please update the style guide with this detail?